go/test/endtoend/backup/vtbackup/backup_only_test.go

```diff
@@ -57,6 +57,8 @@ func TestTabletInitialBackup(t *testing.T) {
 	//    - list the backups, remove them
 	defer cluster.PanicHandler(t)
 
+	waitForReplicationToCatchup([]cluster.Vttablet{*replica1, *replica2})
```

Comment:
Just a reminder that if vtorc is used in these tests, which tablets are replicas is no longer fixed. That's a bigger issue though, not related to this PR.